homework/hw38weatherApp/main.js

```diff
@@ -0,0 +1,47 @@
+const API_ENDPOINT_MAIN = "http://api.openweathermap.org/data/2.5/weather?q=";
+const API_ENDPOINT_END = "&units=metric&APPID=5d066958a60d315387d9492393935c19";
```

Comment:
I don't like it 

You need to create query string method

Read this

https://stackoverflow.com/questions/316781/how-to-build-query-string-with-javascript

https://code-boxx.com/build-query-string-javascript/

Reply:
```
 const params = {
    parameter1: "value_1",
    parameter2: "value_2",
    parameter3: "value_3" 
};

const query = Object
  .keys(params)
  .map(k => encodeURIComponent(k) + '=' + encodeURIComponent(params[k]))
    .join('&');

console.log(query)
```

Reply:
```
const params = new URLSearchParams({
  parameter1: "value_1",
  parameter2: "value_2",
  parameter3: "value_3",
});

console.log(params.toString());
```